src/integrationTest/java/org/openstreetmap/atlas/generator/AtlasGeneratorIntegrationTest.java

```diff
@@ -0,0 +1,82 @@
+package org.openstreetmap.atlas.generator;
+
+import java.io.IOException;
+
+import org.apache.hadoop.fs.Path;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openstreetmap.atlas.exception.CoreException;
+import org.openstreetmap.atlas.generator.tools.streaming.ResourceFileSystem;
+import org.openstreetmap.atlas.streaming.resource.InputStreamResource;
+import org.openstreetmap.atlas.utilities.collections.StringList;
+
+/**
+ * @author matthieun
+ */
+public class AtlasGeneratorIntegrationTest
+{
+    public static final String BOUNDARY = "resource://test/boundaries/DMA.txt";
+    public static final String TREE_13 = "resource://test/sharding/tree-6-13-100000.txt";
+    public static final String TREE_14 = "resource://test/sharding/tree-6-14-100000.txt";
```

Comment:
Is it possible to re-use `tree-6-14-100000.txt` from `src/test/resources/org/openstreetmap/atlas/generator` to avoid having two large identical files?

Reply:
Let me try that.